lib/client.ml

```diff
@@ -7,6 +7,7 @@ type 'a t = {
   state : 'a ref;
   recv_mutex : Lwt_mutex.t;
   state_mutex : Lwt_mutex.t;
+  know_clients : 'a t list;
```

Comment:
Why not Peer.t list?

Reply:
Oh, I don't know why I didn't think about it. Done!